src/main/java/com/programmers/smrtstore/exception/GlobalExceptionHandler.java

```diff
@@ -18,4 +19,10 @@ protected ResponseEntity<ErrorResponse> handleCustomException(
     ) {
         return ErrorResponse.toResponseEntity(e.getErrorCode(), e.getRuntimeValue());
     }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    protected ResponseEntity<String> MethodArgumentNotValidException(Exception e) {
+        return ResponseEntity.status(400)
+            .body(e.getMessage());
```

Comment:
BindingResult 를 따로 빼서 처리하는게 좋아보여요!

```suggestion
public class ValidationErrorResponse {
    private String field;
    private String message;
    private Object rejectedValue;

    // 생성자, getter, setter 등
    public ValidationErrorResponse(String field, String message, Object rejectedValue) {
        this.field = field;
        this.message = message;
        this.rejectedValue = rejectedValue;
    }

    // getter와 setter 메소드...
}

@ExceptionHandler(value = {
        BindException.class,
        MethodArgumentNotValidException.class
})
public ResponseEntity<List<ValidationErrorResponse>> validationError(BindException e) {
    BindingResult bindingResult = e.getBindingResult();

    List<ValidationErrorResponse> errors = new ArrayList<>();
    for (FieldError fieldError : bindingResult.getFieldErrors()) {
        ValidationErrorResponse error = new ValidationErrorResponse(
                fieldError.getField(),
                fieldError.getDefaultMessage(),
                fieldError.getRejectedValue()
        );
        errors.add(error);
    }
    return ResponseEntity.badRequest().body(errors);
}

```